metaflow/plugins/tagging_cli.py

```diff
@@ -0,0 +1,457 @@
+import click
+import sys
+
+from itertools import chain
+
+from metaflow import namespace
+from metaflow.client import Flow, Run
+from metaflow.util import resolve_identity
+from metaflow.exception import CommandException, MetaflowNotFound
+from metaflow.exception import MetaflowNamespaceMismatch
+
+
+def format_tags(obj, system_tags, all_tags, by_tag):
+    hdr = [
+        "",
+        "System tags and *tags*; only *tags* can be modified",
```

Comment:
Perhaps we can remove this specific header and in the footer mention that certain tags are immutable.

Reply:
The idea was to show the ones that could be. I rephrased a bit though.